shared/js/background/https.es6.js

```diff
@@ -196,20 +196,9 @@ class HTTPS {
 
         // Determine host
         const host = utils.extractHostFromURL(reqUrl)
-        const hosts = [host]
 
-        // Check if host has an entry as a wildcarded subdomain
-        const subdomain = utils.extractTopSubdomainFromHost(host)
-        if (subdomain && subdomain !== 'www') {
-            const wildcard = host.replace(subdomain, '*')
-            hosts.push(wildcard)
-        }
-
-        // Check for upgrades
-        for (let i=0; i<hosts.length; i++) {
-            if (this.canUpgradeHost(hosts[i])) {
-                return reqUrl.replace(/^(http|https):\/\//i, 'https://')
-            }
+        if (this.canUpgradeHost(host)) {
+            return reqUrl.replace(/^(http|https):\/\//i, 'https://')
         }
```

Comment:
Previously, we had wildcard rules like `*.foo.com`, which is why we had an extra check for the wildcard version.

We don't have wildcard rules at the moment, so we unnecessarily run an extra linear search for any URLs with subdomains.